release_build_files/readme.md

```diff
@@ -634,7 +634,7 @@ workflow use only during the development of your app, not for publicly shipping
 code.
 
 ## Release Notes
-### Upcoming Changes
+### 9.3.0
 -   Changes
     - General (Android,Linux): Fixed a concurrency bug where waiting for an
       event with a timeout could occasionally return prematurely, as if the
```

Comment:
I know this isn't quite this PR, but is the indentation wrong on the following 2 lines?

Reply:
I think it is correct. All of the later versions also have similar indentation
### 9.0.0
-   Changes
    -   General (iOS): Firebase C++ on iOS is now built using Xcode 13.3.1.

3 spaces between - and Changes, 4 spaces before subsequent dashes

Reply:
Are the lines right below this indented the same? (it looks like they aren't in the GitHub UI)

(the line that reads "timeout had occurred" and the following line, I mean)

Reply:
It still looks consistent to me. 6 spaces of indentation for each line that does not have a hyphen.

You can see the raw value of those lines here if you want:
https://raw.githubusercontent.com/firebase/firebase-cpp-sdk/main/release_build_files/readme.md